docs/www/get.html

```diff
@@ -125,15 +125,15 @@
           <a href="http://www.cpan.org/src/5.0/perl-[% perl_stats.perl_version %].tar.gz"
              class="download" title="Latest Source Tarball"
              >Download Latest Stable Source ([% perl_stats.perl_version %])</a>
-        </p>Consider looking at <a href="https://metacpan.org/release/App-perlbrew/">App::perlbrew</a> to help compile and manage Perl from source</li>
+        </p>Consider looking at <a href="https://metacpan.org/release/App-perlbrew/">App::perlbrew</a> (or <a href="https://github.com/dnmfarrell/berrybrew">Berrybrew</a> for Windows/Strawberry Perl) to help compile and manage Perl from source.</li>
```

Comment:
I don't think we should merge this.  We don't need to list every possible option on this page.

1. Berrybrew belongs on the Strawberry perl download page, not here.
2. It also doesn't build perl from source like Perlbrew does -- so the text is wrong.